python/lib/python_port.c

```diff
@@ -86,12 +99,15 @@ PyObject* py_port_set(PyObject *self, PyObject *args) {
 
     if (val) {
         LF_PRINT_DEBUG("Setting value %p.", val);
-        Py_XDECREF(port->value);
-        Py_INCREF(val);
-        // Call the core lib API to set the port
-        _LF_SET(port, val);
+        python_count_decrement(port->value);
+
+        // 1. Call lf_new_token function to create a new token
```

Comment:
The first two lines of this comment strike me as redundant (remove?), but the third line seems to lack context. Could you elaborate?

Reply:
I assume that it is because the reference count is already set to one when it is created by `PyArg_ParseTuple`, and then no references are saved by this function?